CSharpMath.Rendering/FrontEnd/TextPainter.cs

```diff
@@ -57,6 +59,27 @@ public abstract class TextPainter<TCanvas, TColor> : Painter<TCanvas, TextAtom,
             FontSize, width ?? c.Width,
             c.Height, alignment, padding, offsetX, offsetY
           ));
+        var adjustedCanvasWidth =
+          float.IsInfinity(c.Width) || float.IsNaN(c.Width)
+          ? System.Math.Max(_relativeXCoordDisplay.Displays.CollectionWidth(),
+            _absoluteXCoordDisplay.Displays.IsNonEmpty() ? _absoluteXCoordDisplay.Displays.Max(d => d.Width) : 0)
+          : c.Width;
+        float Δx = 0;
+        var y = float.NegativeInfinity;
+        if ((alignment & (TextAlignment.Left | TextAlignment.Right)) != TextAlignment.Left)
```

Comment:
Just some thoughts:

1. Maybe it would be better to stick to one style (i.e. using braces even in case when `if` is single-line).
2. Looks like `(alignment & (TextAlignment.Left | TextAlignment.Right)` is repeated inside of the `if`. I think it makes sense to create a separate variable.

Reply:
1. Not really sure about requiring braces for one-liners. This feels like an arbitrary restriction on language features. Moreover, [the link between mote braces and less bugs are questionable](https://docs.microsoft.com/en-us/dotnet/api/system.math?view=netcore-3.1). Also, more lines per file will be needed, and the number of brace-lines will be increased, resulting in more noise when reading code.
2. Fixed